testing/testcafe/tests/dataGrid/filterRow.ts

```diff
@@ -34,3 +37,30 @@ test('Filter row\'s height should be adjusted by content (T1072609)', async (t)
 }).after(async () => {
   await changeTheme('generic.light');
 });
+
+test('FilterRow range overlay screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+  const filterEditor = dataGrid.getFilterEditor(1, FilterTextBox);
+
+  await t
+    .click(filterEditor.menuButton);
+  await t
+    .click(filterEditor.menu.getItemByText('Between'))
+    // act
+    .expect(await takeScreenshot('filter-row-overlay', dataGrid.element))
```

Comment:
It's better to add the extension to the screenshot name:

```suggestion
    .expect(await takeScreenshot('filter-row-overlay.png', dataGrid.element))
```